src/test/java/wooteco/subway/controller/StationControllerTest.java

```diff
@@ -40,13 +41,14 @@ public void setUp() {
     @Test
     void createStation() {
         // when
-        ExtractableResponse<Response> response = RestAssured.given().log().all()
-                .body(testStation1)
-                .contentType(MediaType.APPLICATION_JSON_VALUE)
-                .when()
-                .post("/stations")
-                .then().log().all()
-                .extract();
+        ExtractableResponse<Response> response = RestAssured.
+                given().log().all().
+                body(testStation1).
+                contentType(MediaType.APPLICATION_JSON_VALUE).
+                when().
+                post("/stations").
+                then().log().all().
```

Comment:
RestAssured 는 assertThat 을 사용하지 않아도 충분히 검증이 가능한 기능들을 가지고 있어요.

아래 코드와 `org.hamcrest.CoreMatchers` 를 참고해보세요~!

```suggestion
                then().log().all().
                statusCode(HttpStatus.CREATED.value()).
                header("Location", is(not("")));
```